lib/src/arbiter.dart

```diff
@@ -60,3 +60,57 @@ class PriorityArbiter extends Arbiter {
     ]);
   }
 }
+
+/// Arbiter Roun Robin.
+class RoundRobinArbiterlogic extends Arbiter {
+  ///Mask to define pending requests to be attended
+  ///i.e. If bit 4 is granted in a 8 bit request, [requestMask] will be 11110000
```

Comment:
I don't really understand this example, why is bit #4 set to 1 if it's supposed to be cleared?  Maybe a multi-cycle example with how `requestMask` and `grantMask` would help elaborate to readers what these are supposed to do.